<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frm_PG_AssessmentResult_Add</class>
 <widget class="QDialog" name="Frm_PG_AssessmentResult_Add">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>655</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>添加/编辑毁伤结果</string>
  </property>
  <widget class="QGroupBox" name="gb_association_info">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>631</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>关联信息</string>
   </property>
   <widget class="QLabel" name="lbl_scene_id">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>25</y>
      <width>49</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>场景ID：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_select_scene">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>25</y>
      <width>75</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>选择场景</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_parameter_id">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>25</y>
      <width>49</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>参数ID：</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_ammunition_id">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>70</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>弹药名称：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ed_ammunition_id">
    <property name="geometry">
     <rect>
      <x>72</x>
      <y>70</y>
      <width>240</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>background-color: #f0f0f0;</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_ammunition_note">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>70</y>
      <width>56</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;font color='gray'&gt;(自动填充)&lt;/font&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_target_type">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>110</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>目标类型：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmb_target_type">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>72</x>
      <y>110</y>
      <width>240</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>background-color: #f0f0f0;</string>
    </property>
    <item>
     <property name="text">
      <string>机场跑道</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>单机掩蔽库</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>地下指挥所</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="lbl_target_type_note">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>110</y>
      <width>56</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;font color='gray'&gt;(自动填充)&lt;/font&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_target_id">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>150</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>目标名称：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ed_target_id">
    <property name="geometry">
     <rect>
      <x>72</x>
      <y>150</y>
      <width>240</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>background-color: #f0f0f0;</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_target_id_note">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>150</y>
      <width>56</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;font color='gray'&gt;(自动填充)&lt;/font&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_calculate_damage">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>120</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>35</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string>QPushButton {
    background-color: #4CAF50;
    color: white;
    font-weight: bold;
    border-radius: 5px;
}
QPushButton:hover {
    background-color: #45a049;
}
QPushButton:pressed {
    background-color: #3d8b40;
}</string>
    </property>
    <property name="text">
     <string>计算毁伤结果</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_select_param">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>25</y>
      <width>75</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>选择参数</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ed_parameter_id">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>25</y>
      <width>132</width>
      <height>25</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="ed_scene_id">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>25</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_required_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>25</y>
      <width>13</width>
      <height>23</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>13</width>
      <height>23</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>13</width>
      <height>23</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgba(255, 0, 0, 255)</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_required_3">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>25</y>
      <width>13</width>
      <height>23</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>13</width>
      <height>23</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>13</width>
      <height>23</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgba(255, 0, 0, 255)</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gb_crater_params">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>631</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>弹坑参数</string>
   </property>
   <widget class="QLabel" name="lbl_depth">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>38</y>
      <width>80</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>弹坑深度(m)：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ed_depth">
    <property name="geometry">
     <rect>
      <x>101</x>
      <y>38</y>
      <width>110</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_diameter">
    <property name="geometry">
     <rect>
      <x>217</x>
      <y>38</y>
      <width>80</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>弹坑直径(m)：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ed_diameter">
    <property name="geometry">
     <rect>
      <x>303</x>
      <y>38</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_volume">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>38</y>
      <width>85</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>弹坑容积(m³)：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ed_volume">
    <property name="geometry">
     <rect>
      <x>511</x>
      <y>38</y>
      <width>110</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_area">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>77</y>
      <width>85</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>弹坑面积(m²)：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ed_area">
    <property name="geometry">
     <rect>
      <x>101</x>
      <y>77</y>
      <width>110</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_length">
    <property name="geometry">
     <rect>
      <x>217</x>
      <y>77</y>
      <width>80</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>弹坑长度(m)：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ed_length">
    <property name="geometry">
     <rect>
      <x>303</x>
      <y>77</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_width">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>77</y>
      <width>80</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>弹坑宽度(m)：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ed_width">
    <property name="geometry">
     <rect>
      <x>511</x>
      <y>77</y>
      <width>110</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gb_damage_assessment">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>631</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>毁伤评估</string>
   </property>
   <widget class="QLabel" name="lbl_discturction">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>结构破坏程度：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ed_discturction">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>201</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_damage_degree">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>毁伤等级：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmb_damage_degree">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>20</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>未达到轻度毁伤</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>轻度毁伤</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>中度毁伤</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>重度毁伤</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>完全摧毁</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_save">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>410</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cancel">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>410</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
